platform/broadcom/sonic-platform-modules-inventec/d7332/modules/pmbus.h

```diff
@@ -0,0 +1,466 @@
+/*
+ * pmbus.h - Common defines and structures for PMBus devices
```

Comment:
in kernel, why do we need it.

Reply:
pmbus.h is a free software, and we modified it to add fan read/write functions to fulfill customer requirement